.circleci/config.yml

```diff
@@ -0,0 +1,80 @@
+version: 2.1
+orbs:
+  android: circleci/android@2.0
```

Comment:
can we update this to 
  `circleci/android@2.1.2`. been using this is places repo